spec/base_spec.rb

```diff
@@ -451,13 +478,31 @@ def fixup_aggregate_schema(loaded_version)
         expect(query_response.models.sort.map(&:to_store)).to eq([dummy_model1, dummy_model2].sort.map(&:to_store))
       end
 
+      it "filters across multiple indices if provided" do
+        clear_and_create_index(index: DummyUniqueIndexModel.index_name)
+        DummyElasticSearchModel.create!(my_string: "Hello", my_other_string: "Goodbye")
+        DummyUniqueIndexModel.create!(my_string: "Hello")
```

Comment:
In order to ensure the queries are working as expected, you should add documents for both indices that should be excluded from the query response
e.g.
```
DummyElasticSearchModel.create!(my_string: "Exclude this one")
DummyUniqueIndexModel.create!(my_string: "Exclude this one too")
```